CHANGELOG.md

```diff
@@ -14,31 +16,53 @@ might need to be adjusted and either explicitly name the positional
 argument or resolve the type ambiguity by casting to the intended type.
 
 **Features**
+* PG15 support added
+* MultiNode High Avalibility improvements
+* #4335 Show warnings when not following best practices
+* #4476 Batch rows on access node for distributed COPY
+* #4562 Exponentially backoff when out of background workers
 * #4650 Show warnings when not following best practices
+* #4664 Introduce fixed schedules for background jobs
+* #4668 Hierarchical Continuous Aggregates
 * #4670 Add timezone support to time_bucket_gapfill
+* #4678 Add interface for troubleshooting job failures
 * #4718 Add ability to merge chunks while compressing
 * #4786 Extend the now() optimization to also apply to CURRENT_TIMESTAMP
+* #4820 Support parameterized data node scans in joins
+* #4830 Add function to alter data nodes
 
 **Bugfixes**
+* #4663 Don't error when compression metadata is missing
 * #4673 Fix now() constification for VIEWs
 * #4681 Fix compression_chunk_size primary key
 * #4696 Report warning when enabling compression on hypertable
 * #4745 Fix FK constraint violation error while insert into hypertable which references partitioned table
 * #4756 Improve compression job IO performance
+* #4770 Continue compressing other chunks after an error
+* #4794 Fix degraded performance seen on timescaledb_internal.hypertable_local_size() function
 * #4807 Fix segmentation fault during INSERT into compressed hypertable
-* #4840 Fix performance regressions in the copy code
+* #4822 Fix missing segmentby compression option in CAGGs
 * #4823 Fix a crash that could occur when using nested user-defined functions with hypertables
+* #4840 Fix performance regressions in the copy code
+* #4860 Dist analyze: cannot insert multiple commands into a prepared statement
 * #4898 Fix cagg migration failure when trying to resume
+* #4904 Remove BitmapScan support in DecompressChunk
 * #4906 Fix a performance regression in the query planner by speeding up frozen chunk state checks
 * #4910 Fix a typo in process_compressed_data_out
+* #4915 Fix assetion (num_found == 0) hit in ts_bgw_job_find_with_lock
```

Comment:
We usually dont include assertion fixes here since they are internal
```suggestion
```

Reply:
removed